Makefile

```diff
@@ -325,6 +325,7 @@ lint-fix: ##@test Run code style checks and fix issues
 	ALL_CLOJURE_FILES=$(call find_all_clojure_files) && \
 	zprint '{:search-config? true}' -sw $$ALL_CLOJURE_FILES && \
 	zprint '{:search-config? true}' -sw $$ALL_CLOJURE_FILES && \
+	clojure-lsp --ns-exclude-regex ".*/src/status_im2/core\.cljs$$" clean-ns && \
```

Comment:
`--ns-exclude-regex` is not what it looks like
it's actually lint against file URL like this `file:///Users/yqrashawn/workspace/office/status-mobile/src/status_im/ui/screens/profile/contact/views.cljs`

should create an issue for clojure-lsp